lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -7433,6 +7317,7 @@ AdjointValue PullbackEmitter::makeAggregateAdjointValue(
   return AdjointValue::createAggregate(allocator, remapType(type), elements);
 }
 
+[[deprecated]]
```

Comment:
I’d suggest removing all deprecated unused functions because they are going to be untested from now on.

Reply:
How about deleting all `AdjointValue` construction/materialization/accumulation utilities, except for the definition of `AdjointValue `itself? `AdjointValue` itself can be repurposed for `AdjointBuffer` symbolic zero optimization.

Reply:
Done in https://github.com/apple/swift/pull/26709/commits/82c775f10a1dadc41df702e31fa4d47d0f6c6469. Minus 500+ lines.

Reply:
That sounds fine. We'll probably want to rename it to `SymbolicTangent` or something.